vernac/g_vernac.mlg

```diff
@@ -492,6 +492,10 @@ let warn_deprecated_include_type =
   CWarnings.create ~name:"deprecated-include-type" ~category:"deprecated"
          (fun () -> strbrk "Include Type is deprecated; use Include instead")
 
+let warn_deprecated_as_ident_kind =
+  CWarnings.create ~name:"deprecated-as-ident-kind" ~category:"deprecated"
+         (fun () -> strbrk "grammar kind \"as ident\" was formerly accepting any identifier or \"_\"; it now accepts only identifiers; use \"as name\" if you expect \"_\" to be parsed too, or silence the warning if you actually intended to accept only identifiers.")
```

Comment:
```suggestion
         (fun () -> strbrk "grammar kind \"as ident\" was formerly accepting any identifier or \"_\"; in the future, it will only accept identifiers; use \"as name\" if you expect \"_\" to be parsed too, or silence the warning if you actually intended to accept only identifiers.")
```

Reply:
In the `as ident` case (compared to the `x ident`, this is really "now", on the assumption that few users have used it yet since it is not too old (v8.8) and of relatively marginal purprose.